Packs/IdentityRecordedFuture/Integrations/IdentityRecordedFuture/IdentityRecordedFuture.yml

```diff
@@ -0,0 +1,184 @@
+commonfields:
+  id: Recorded Future Identity
+  version: -1
+name: Recorded Future Identity
+display: Recorded Future Identity
+category: Data Enrichment & Threat Intelligence
+image:
+description: Recorded Future Identity Integration that provides access to Recorded
+  Future Identity module data.
+configuration:
+- display: Server URL (e.g., https://api.recordedfuture.com/gw/xsoar-identity)
+  name: server_url
+  defaultvalue: https://api.recordedfuture.com/gw/xsoar-identity
+  type: 0
+  required: true
+- display: API Token
+  name: token
+  type: 4
+  required: true
+- display: Trust any certificate (not secure)
+  name: unsecure
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: Password properties
+  name: password_properties
+  defaultvalue: Letter,Number
+  type: 16
+  required: false
+  options:
+  - Letter
+  - Number
+  - Symbol
+  - UpperCase
+  - LowerCase
+  - MixedCase
+  - AtLeast8Characters
+  - AtLeast12Characters
+  - AtLeast16Characters
+  - AtLeast24Characters
+  additionalinfo: Filter for password properties
+- display: Limit of identities to get (min is 0 and max is 10 000)
+  name: limit_identities
+  defaultvalue: "500"
+  required: false
+  type: 0
+- display: Domains list separated by semicolon. (e.g. mycompany.com; othercompany.com )
+  name: domains
+  type: 12
+  required: true
+  additionalinfo: List of domains.
+script:
+  script: '-'
+  type: python
+  subtype: python3
+  dockerimage: demisto/python3:3.10.1.25933
+  commands:
+  - name: recordedfuture-identity-search
+    description: Search for identities in Recorded Future Identity Dataset
+    arguments:
+    - name: latest-downloaded
+      required: false
+      auto: PREDEFINED
+      defaultValue: "All time"
+      description: Time period for last downloaded time.
+      predefined:
+      - "Last 24 Hours"
+      - "Last 7 Days"
+      - "Last Month"
+      - "Last 3 Months"
+      - "Last 6 Months"
+      - "Last Year"
+      - "All time"
```

Comment:
Does the API only support these values? or can any period be queried?

Reply:
API support any date but to make the User Interface easier to use we decided to have periods predefined.

Reply:
Our recommendation is to allow a flexible querying when the server supports it: use a free text (type 0) argument field, and call dateparser.parse on it. This subtracts the input from the current time and returns a `datetime.datetime` object:

```python
>>> parse('3 years')
datetime.datetime(2019, 1, 26, 14, 33, 37, 196384)
```

"all time" is the only thing that won't be automatically parsed by dateparser - to go around it, it's possible that you consider an empty input as all times, and check for it before calling `dateparser.parse`. Just make sure to tell the users (in either the `additional info` or `display` fields of the argument, in the yaml) to make it easier for them to use it.